### PR TITLE
Saved content folder title rename

### PR DESCRIPTION
**Pull request by chiruzzimarco**
Related to issue [#3803](https://github.com/jazkarta/mountaineers/issues/3803)

### CODE REVIEW

**Comment by chiruzzimarco** [comment]
@alecpm I was quite confused by the conversation with Jeff on the ticket. Different approaches were discussed and I hope I got it right.

This PR will display the action title set by the user in place of the folder title. If the action have no title set it generate the title from the containing form name. This won't actually change the folder title/name/url, but I think that's enough to match Jeff request.

**Comment by alecpm** [comment]
@chiruzzimarco This is not correct, the view should include the parent form title in the title, so something like `${form_title}: ${folder_title}`.

**Comment by alecpm** [comment]
@chiruzzimarco See my comment above. I think including the form title is important here. Using the action title here is probably reasonable too.

**Comment by alecpm** [comment]
@chiruzzimarco I think you're correct. We only need the action name. Using the containing form name was an option Jeff provided, but using the action title as you do here is probably better. So no changes needed except to make it compatible with master.

File: src/jazkarta/easyformplugin/savecontent/browser.py [hunk 1]
```diff
@@ -35,3 +35,9 @@ def updateWidgets(self):
 
 class FormContentFolderListingView(FolderView):
     """Form submissions listing page with a customized template"""
+
+    def getTitle(self):
```
Comment (alecpm) [inline src/jazkarta/easyformplugin/savecontent/browser.py]: Why not just override `Title()` here? That's the standard method, and should avoid the need for the listing customization.

**Review by alecpm — CHANGES_REQUESTED** [review]
I think you should just override the standard `Title()` accessor here, unless that cause problems.

File: src/jazkarta/easyformplugin/savecontent/browser.py (hunk 1 above)
Comment (chiruzzimarco) [reply src/jazkarta/easyformplugin/savecontent/browser.py]: @alecpm I renamed the `getTitle` method to `Title` for consistency, but we'll still need the listing template customization.

Turns out that we actually need also the `content-core` block (that was removed [in this PR](https://github.com/jazkarta/jazkarta.easyformplugin.savecontent/pull/2/files#r1071597272)) to be customized due to [this change](https://github.com/jazkarta/jazkarta.easyformplugin.savecontent/pull/4/files#diff-ebb959d44f169428a1831f43fef00446ce9e546ff285bead956540512980e866R44) related to [this issue](https://github.com/jazkarta/mountaineers/issues/3803).

**Review by alecpm — CHANGES_REQUESTED** [review]
You should use: 

```
<metal:content-core fill-slot="content-core">
        <metal:block use-macro="context/@@listing_view/macros/content-core" />
</metal:content-core>
```

instead of your current `content-core` code, unless/until you have customizations for the listing.

**Pull request merged**